Add a new TiDB Cloud Integration (#1095)
DataDog/integrations-extras: tidb_cloud/README.md

# TiDB Cloud Integration

## Overview

> [TiDB Cloud][1] makes deploying, managing and maintaining your TiDB clusters even simpler with a fully managed cloud instance that you control through an intuitive console. 
> 
> TiDB Cloud allows developers and DBAs with little or no training to handle once-complex tasks such as infrastructure management and cluster deployment with ease, to focus on your applications, not the complexities of your database.
> 
> The Datadog integration is an observability feature of TiDB Cloud that exports metrics of TiDB Cloud clusters to your Datadog platform.

Reviewer: ```suggestion
[TiDB Cloud][1] is a fully managed cloud service of TiDB, an open-source database.

Use the TiDB Cloud Datadog integration to export metrics from TiDB Cloud clusters to Datadog.
```

Reviewer: Sorry for removing your writing, but we don't allow marking-type prose in the integrations docs.

> **Note:**
>
> - For TiDB clusters on premises, please refer to [TiDB Integration][4].

## Setup

To set up the TiDB Cloud Datadog integration for your cluster, provide a Datadog API key and region to TiDB Cloud.

> Refer to the [TiDB Cloud Preferences Page][2] to configure the Datadog integration for your TiDB Cloud project.

Reviewer: ```suggestion
See [TiDB Cloud Preferences][2] to configure the Datadog integration for your TiDB Cloud project.
```
